src/main/java/org/testng/annotations/Parameters.java

```diff
@@ -14,7 +13,7 @@
  * @author <a href="mailto:cedric&#64;beust.com">Cedric Beust</a>
  */
 @Retention(RUNTIME)
-@Target({METHOD, CONSTRUCTOR, TYPE })
+@Target({METHOD, TYPE})
 public @interface Parameters {
```

Comment:
Any specific reason as to why the `CONSTRUCTOR` target is being removed ? Currently TestNG  supports parameter injection during instantiation from a TestNG suite xml file. I am including a sample below which shows what I am referring to. This change will break this functionality.

```java
import org.testng.Assert;
import org.testng.annotations.Parameters;
import org.testng.annotations.Test;

import java.util.Calendar;
import java.util.Date;

public class TestClass {
    private Date actual;

    @Parameters({"day", "month", "year"})
    public TestClass(int day, int month, int year) {
        Calendar actualInstance = Calendar.getInstance();
        actualInstance.set(year, month, day);
        actual = actualInstance.getTime();
    }

    @Test
    public void testMethod() {
        Calendar expectedInstance = Calendar.getInstance();
        expectedInstance.set(2017, 1, 1);
        Date expected = expectedInstance.getTime();
        Assert.assertTrue(expected.after(actual));
    }
}
```

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite name="pr_1488_Suite" verbose="2">
    <test name="pr_1488_test">
        <parameter name="day" value="1"/>
        <parameter name="month" value="1"/>
        <parameter name="year" value="2016"/>
        <classes>
            <class name="com.rationaleemotions.github.pr1488.TestClass"/>
        </classes>
    </test>
</suite>
```